content/en/logs/log_configuration/rehydrating.md

```diff
@@ -185,6 +182,18 @@ In order to rehydrate log events from your archives, Datadog uses a service acco
 {{% /tab %}}
 {{< /tabs >}}
 
+
+## Understanding Rehydration scan sizes
+
+The query is applied _after_ the files matching the time period are downloaded from your archive. As a result, the rehydration scan size is based on **total volume of logs retrieved from the archive**, not just the number of logs matching the query. Archive storage is time-base, so queries scoped to specific filters (such as `service:A`) still retrieve all logs within the selected time window, including those from other services (such as `service:A` and `service:B`).
+
+Because query filters are applied post-download, reducing the date range is the most effective way to limit scan size and reduce cloud data transfer costs.
```

Comment:
```suggestion
Reducing the date range is the most effective way to limit scan size and minimize cloud data transfer costs, because query filters are applied after data is downloaded
```